src/core/scrobbler/webhook-scrobbler.ts

```diff
@@ -100,15 +94,21 @@ export default class WebhookScrobbler extends BaseScrobbler<'Webhook'> {
 			body: JSON.stringify(request),
 		};
 
-		const promise = fetch(this.userApiUrl, requestInfo);
-
+		const promises: Promise<Response>[] = [];
+		for (const props of this.arrayProperties) {
+			promises.push(fetch(props.userApiUrl, requestInfo));
+		}
 		const timeout = this.REQUEST_TIMEOUT;
 
-		let response = null;
 		try {
-			response = await timeoutPromise(timeout, promise);
-			if (response.status !== 200) {
-				return ServiceCallResult.ERROR_OTHER;
+			const responses = await timeoutPromise(
+				timeout,
+				Promise.all(promises),
+			);
+			for (const response of responses) {
+				if (response.status !== 200) {
```

Comment:
Should we add a log line here to aid in debugging what requests fail or do you think the network tab is clear enough?

Reply:
Agreed, added.